{{cookiecutter.project_slug}}/static/src/js/base.js

```diff
@@ -1,6 +1,6 @@
 const main = document.querySelector('.main');
 
-// Stretch main to fill height of screen minus header and footer
+// Stretch main height to fill the screen
 if (main) {
     const footer_height = document.querySelector('.footer').offsetHeight;
     const header_height = document.querySelector('.header').offsetHeight;
```

Comment:
Might we be better off doing this with flexbox or css grid? if we set body to `min-height: 100vh;` `display: flex;` `flex-dir: column;` and set `<main>` to have `flex-grow: 1;` we can avoid the need for JS.

Reply:
possibly, yes! Make a PR for that when you get a chance and I'll review. This is just what we've used in the past and works well, it's not the worst thing to lose if JS is playing up, but if there's a nice sass solution lets do that!

Reply:
Have added a card linking to the comment in to the frontend improvements backlog. https://favro.com/organization/b03d6d6d9b11b61167ac4246/418c53a770389640d415a230?card=Dev-12017